<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Displayer</class>
 <widget class="QWidget" name="Displayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 7, 56);
background-color: rgb(107, 255, 62);
background-color: rgb(164, 133, 255);</string>
  </property>
  <widget class="QLabel" name="Image">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>30</y>
     <width>191</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Savebutton">
   <property name="geometry">
    <rect>
     <x>622</x>
     <y>568</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 255, 113);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Imagebutton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 255, 113);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Choose Image path</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangeGroup_PasswordButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>410</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 255, 113);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteStudent_teacherButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>540</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources for SMS/removeitem_resized.jpeg);
background-position: center;
background-repitition: no-repitition;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>161</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="ID">
      <property name="styleSheet">
       <string notr="true">color: rgb(16, 16, 15);</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Name">
      <property name="styleSheet">
       <string notr="true">color: rgb(16, 16, 15);</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Email">
      <property name="styleSheet">
       <string notr="true">color: rgb(16, 16, 15);</string>
      </property>
      <property name="text">
       <string>E-mail</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Phone_number">
      <property name="styleSheet">
       <string notr="true">color: rgb(16, 16, 15);</string>
      </property>
      <property name="text">
       <string>Phone Number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="YearandGroup_Password">
      <property name="styleSheet">
       <string notr="true">color: rgb(16, 16, 15);</string>
      </property>
      <property name="text">
       <string>Year/Group</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>449</x>
     <y>91</y>
     <width>271</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="IDedit">
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Nameedit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="Emailedit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="Phone_numberedit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="YearandGroup_passwordedit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
